src/update/attributes.rs

```diff
@@ -256,8 +311,15 @@ impl PathAttribute {
                 Ok(PathAttribute::AS4_AGGREGATOR((asn, ip)))
             }
             20 => {
-                stream.read_u16::<BigEndian>()?;
-                let ip = Ipv4Addr::from(stream.read_u32::<BigEndian>()?);
+                let mut buf = vec![0u8; length as usize];
+                stream.read_exact(&mut buf)?;
+
+                let mut cur = Cursor::new(buf);
+                let _ = cur.read_u16::<BigEndian>()?;
+                // I have no idea what this is.. both Junos and IOS-XR send this but it's
+                // not covered in the RFC at all
+                let _ = cur.read_u64::<BigEndian>()?;
```

Comment:
That's very interesting haha :) Let's create an issue of this after we merge the PR.